CMakeLists.txt

```
@@ -41,6 +37,17 @@ option(OTIO_AUTOMATIC_SUBMODULES "Fetch submodules automatically" ON)
 # Set option dependent variables
 
 if(OTIO_PYTHON_INSTALL)
+    # Find the python interpreter using FindPython introduced in CMake 3.12.
+    # The Development.Module component is used instead of the Development one
```

```suggestion
    # The Development.Module component introduced in CMake 3.18.2 is used instead of the Development one
```
I suggest documenting this point, if I've understood it properly, since you changed the minimum version to 3.18.2, but mention 3.12 in the comment here.

---

Actually, `Development.Module` was introduced in 3.18.0 but pybind11 recommends 3.18.2+. I'll fix that and document that pybind11 recommends 3.18.2+.

---

@meshula I adapted the comment and updated the comment for the `cmake_minimum_required`, which should hopefully be enough. Let me know if it's clearer.